docs/README.md

```diff
@@ -1,61 +1,16 @@
-- If you are not a developer at all, we have [an Editors-Marketers getting started guide.](https://academy.jahia.com/tutorials-get-started/editorsmarketers/adding-a-new-page)
-
-This documentation is meant for self-learning, but we also offer workshops and training sessions with our team. If you are interested in that, please [contact us.](https://www.jahia.com/contact)
-
-If you're still reading this and want to get started—for real—let's dive in!
-
-> [!IMPORTANT]
-> This documentation is still in writing and some parts might be incomplete or missing. We are working hard to provide you with the best possible content. If you have any feedback or suggestions, [please let us know!](https://github.com/Jahia/javascript-modules/issues/new?template=other.md)
-
-## Getting started
-
-The goal of this section is to help you create a Jahia website with React and JavaScript from scratch. This a learn-by-doing tutorial that guides you through the integration of a Jahia website.
-
-Start here: [Setting up your dev environment](./1-getting-started/1-dev-environment/)
-
-## Guides
-
-This section contains independent, self-contained guides that help you build specific features or solve specific problems. Each guide is a step-by-step tutorial that you can follow to achieve a specific goal.
-
-- [Building a Feedback Form](./2-guides/1-building-a-feedback-form/)
-- [Island Architecture](./2-guides/2-island-architecture/)
-
-<!-- Hidden until available:
-- [Accessibility and Performance](./2-guides/2-accessibility-and-performance/)
-- [Building a Menu and Sitemap](./2-guides/3-building-a-menu-and-sitemap/)
-- [Adding Icons](./2-guides/4-adding-icons/)
-- [Adding Tailwind](./2-guides/5-adding-tailwind/)
-- [Building for Production](./2-guides/6-building-for-production/)
-- [Rendering Markdown](./2-guides/7-rendering-markdown/)
-- [RSS Feed](./2-guides/8-rss-feed/)
-- [Debugging](./2-guides/9-debugging/)
-- [Using Web Components](./2-guides/A-using-web-components/)
-
--->
-
-More guides are coming soon, and contributions are welcome!
-
-## Reference
-
-This section is meant to give an exhaustive overview of the features and concepts of JavaScript Modules. It is a reference guide that you can use to look up specific information or deepen your understanding of a particular topic.
-
-- [CND format](./3-reference/1-cnd-format/)
-
-<!-- Hidden until available:
-- [Tools](./3-reference/2-tools/)
-- [JCR](./3-reference/3-jcr/)
-- [Mixins](./3-reference/4-mixins/)
-- [Children and Area](./3-reference/5-children-and-area/)
-- [Nav Builder](./3-reference/6-nav-builder/)
-- [UI Extensions](./3-reference/7-ui-extensions/)
-- [Single Directory Components and Resources](./3-reference/8-single-directory-components-and-resources/)
-- [Java Interoperability](./3-reference/9-java-interoperability/)
-- [SSR and Hydration](./3-reference/A-ssr-and-hydration/)
--->
+---
+page:
+  $path: /sites/academy/home/get-started/tutorials-home
```

Comment:
Does this page really belong to the JavaScript Modules repo?
Why not keeping [the existing one](https://edit.jahia.com/jahia/jcontent/academy/en/pages/home/get-started/tutorials-home)?